src/openms/include/OpenMS/FORMAT/FileHandler.h

```diff
@@ -130,24 +158,33 @@ namespace OpenMS
     */
     static FileTypes::Type getTypeByContent(const String& filename);
 
-    /// Returns if the file type is supported in this build of the library
+    /// @brief Returns if the file type is supported in this build of the library
     static bool isSupported(FileTypes::Type type);
 
-    /// Mutable access to the options for loading/storing
+    /// @brief Mutable access to the options for loading/storing
     PeakFileOptions& getOptions();
 
-    /// Non-mutable access to the options for loading/storing
+    /// @brief Non-mutable access to the options for loading/storing
     const PeakFileOptions& getOptions() const;
 
-    /// set options for loading/storing
+    /// @brief Mutable access to the feature file options for loading/storing
+    FeatureFileOptions& getFeatOptions();
+
+    /// @brief Non-mutable access to the feature file options for loading/storing
+    const FeatureFileOptions& getFeatOptions() const;
+
+    /// @brief set options for loading/storing
     void setOptions(const PeakFileOptions&);
 
+    /// @brief set feature file options for loading/storing
+    void setFeatOptions(const FeatureFileOptions&);
+
     /**
       @brief Loads a file into an MSExperiment
 
       @param filename The file name of the file to load.
       @param exp The experiment to load the data into.
-      @param force_type Forces to load the file with that file type. If no type is forced, it is determined from the extension (or from the content if that fails).
+      @param allowed_types a FileTypeList containing the types of files to support. If not empty the extension, and failing that the contents are checked to make sure they match
```

Comment:
doesn't document semantics for allowed_types.emtpy()

Reply:
It does now.